tasks/README.md

```diff
@@ -114,6 +115,40 @@ Returns all the completed tasks of user if query `completed=true` is passed, els
     - **Content:** `{ 'statusCode': 404, 'error': 'Not Found', 'message': 'User doesn't exist' }`
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+    
+    
+## **GET /tasks/paginated**
+
+Returns all the tasks which are created after the ```<task_object>``` whose "id" would be passed in the query ```after```. The no. of returned documents is controlled by ```limit```again from the query which will also have a default.
+
+- **Params**  
+  None
+- **Query**  
+  _Required:_ limit=[number], after=[string(task id)]
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  None
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+```
+{
+  tasks : {<task_object>}[],
+  total : {<task_object>}[] length,
+  till : <task_object_id>
```

Comment:
Great idea!
Should we combine `total`, `till` etc under a common field in the result?

If you ask me, I would highly recommend us adopting: https://jsonapi.org/ spec